prac_05/hex_colours.py

```diff
@@ -1,5 +1,5 @@
 """
-CP1404 HEX COLOURS
+CP1404 HEX_COLOURS
 By Samantha Martin
 """
 
```

Comment:
Line 15 can be written as a while loop with the if/else statement inside it and the for loop shouldn't be needed.